roles/pulp_database/vars/Fedora-35.yml

```diff
@@ -0,0 +1 @@
+Fedora-34.yml
```

Comment:
Is this a badly disguised symlink?

Reply:
yep, it is!

Reply:
I thought they'd done better in the past.

Reply:
How am i supposed to see this is not a file containing the name of another file? Right now, here i can only assume the tests would catch this ambiguity.